pkgs/by-name/mi/mistral-rs/package.nix

```diff
@@ -24,7 +24,7 @@
   nix-update-script,
 
   cudaPackages,
-  cudaCapability ? null,
+  cudaCapability ? config.cudaCapability,
```

Comment:
H'm haven't heard about `config.cudaCapability` (singular)

Reply:
Indeed, and `config.cudaCapabilities` is already used in the current implementation.
I removed this change.